sdk/translation/ai-translation-text/package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "name": "@azure-rest/ai-translation-text",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-beta.1",
+  "description": "Microsoft Translator Service",
+  "keywords": [
+    "node",
+    "azure",
+    "cloud",
+    "typescript",
+    "browser",
+    "isomorphic",
+    "translate",
+    "translation"
+  ],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/ai-translation-text.d.ts",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/ai-translation-text.d.ts",
+    "README.md",
+    "LICENSE",
+    "review/*"
```

Comment:
do we need to ship the review folder?

Reply:
Removed

Reply:
I don't see it removed yet so I will mark this back as unresolved for now.

Reply:
Never mind, I opened the wrong commit.